test_helpers/clean_feature_context.go

```diff
@@ -49,6 +90,74 @@ func CleanFeatureContext(s *godog.Suite) {
 		return killTestContainers(dockerComposeDir, appName)
 	})
 
+	s.Step(`^my machine has running application and test containers$`, func() error {
+		appContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", appNetwork, "up")
+		appContainerProcess.SetDir(path.Join(appDir, "tmp"))
+		err := appContainerProcess.Start()
+		if err != nil {
+			return err
+		}
+		testContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", testNetwork, "up")
+		testContainerProcess.SetDir(path.Join(appDir, "service", "tests", "tmp"))
+		err = testContainerProcess.Start()
+		time.Sleep(2500 * time.Millisecond)
```

Comment:
Why is this sleep needed?

Reply:
To give the containers time to "start up". I think there was a point in the test between when `docker-compose up` has been executed and the actual containers show up as running in `docker ps`

Reply:
Can we wait for them to show up in `docker ps` then? I'd prefer we have deterministic waits instead of a time we hope is long enough